srcpkgs/starship/template

```diff
@@ -11,8 +11,20 @@ maintainer="Aluísio Augusto Silva Gonçalves <aluisio@aasg.name>"
 license="ISC"
 homepage="https://starship.rs"
 distfiles="https://github.com/starship/starship/archive/v${version}.tar.gz"
-checksum=ce2c86e3f3a7d8cd2e1a204e7ef491473fdd93c41bb43a4718fdd66039f7bc99
+checksum=cf7b5848949bc1e61dc771ef2a429e4ccb8d339e1ca705bdf521fc4a4fc96309
 
 post_install() {
+	if ! [ "$CROSS_BUILD" ]; then
+		# generate shell completions
+		ln -s target/${RUST_TARGET}/release/starship starship
+		./starship completions zsh > starship.zsh
```

Comment:
I think we can pull `starship` in `hostdepends`
for cross build and building completion for cross.

Reply:
Good idea, thanks!

Reply:
@sgn this is probably even more useful for something like rustup. It could make sense to add it there as well.